youtube_dl/extractor/pr0gramm.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from datetime import datetime
+import re
+from ..utils import (
+    merge_dicts
+)
+
+
+class Pr0grammStaticIE(InfoExtractor):
+    # https://pr0gramm.com/static/5466437
+    _VALID_URL = r'https?://pr0gramm\.com/static/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://pr0gramm.com/static/5466437',
+        'md5': '52fa540d70d3edc286846f8ca85938aa',
+        'info_dict': {
+            'id': '5466437',
+            'ext': 'mp4',
+            'title': 'pr0gramm',
+            'uploader': 'g11st',
+            'upload_date': '20221221',
+            'release_date': '20221221',
+            'timestamp': 1671580800,
+            'release_timestamp': 1671580800
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        # Fetch media sources
+        entries = self._parse_html5_media_entries(url, webpage, video_id)
+        media_info = entries[0]
+
+        # this raises if there are no formats
+        self._sort_formats(media_info.get('formats') or [])
+
+        # Fetch author
+        uploader = self._html_search_regex(r'by\W+([\w-]+)\W+', webpage, 'uploader')
+
+        # Fetch approx upload timestamp from filename
+        # Have None-defaults in case the extraction fails
+        uploadDay = None
+        uploadMon = None
+        uploadYear = None
+        uploadTimestamp = None
+        uploadTimestr = None
+        # (//img.pr0gramm.com/2022/12/21/62ae8aa5e2da0ebf.mp4)
+        m = re.search(r'.*//img\.pr0gramm\.com/(?P<year>[0-9]+)/(?P<mon>[0-9]+)/(?P<day>[0-9]+)/\w+\.\w{,4}.*', webpage)
+
+        if (m):
+            # Up to a day of accuracy should suffice...
+            uploadDay = m.groupdict().get('day')
+            uploadMon = m.groupdict().get('mon')
+            uploadYear = m.groupdict().get('year')
+            uploadDatetime = datetime(int(uploadYear), int(uploadMon), int(uploadDay))
+            epochDatetime = datetime(1970, 1, 1)
+            uploadTimestamp = (uploadDatetime - epochDatetime).total_seconds()
+            uploadTimestr = str(uploadYear) + str(uploadMon) + str(uploadDay)
```

Comment:
As you only have a date, don't set a misleading `timestamp`, but just `upload_date`: you may need to adjust the test cases. To save more characters say `\d` instead of `[0-9]`. Also, in a `search`, there's no need to match the start and end with wildcards.
```suggestion
        # Have None-default in case the extraction fails
        uploadTimestr = None
        # (//img.pr0gramm.com/2022/12/21/62ae8aa5e2da0ebf.mp4)
        m = re.search(r'//img\.pr0gramm\.com/(?P<date>[0-9]{4}/[0-9]{2}/[0-9]{2})/\w+\.\w{,4}', webpage)

        if m:
            # Up to a day of accuracy should suffice...
            uploadTimestr = m.group('date').replace('/', '')
```